### PR TITLE
docs: Adding the json Schema to the documentation

### PR DESCRIPTION
**Pull request by Rajni2002**
fix https://github.com/excalidraw/excalidraw/issues/6804

### CODE REVIEW

File: dev-docs/docs/codebase/json-schema.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# JSON Schema
+
+```json
+{
+    // Schema information
+    "type": "excalidraw",
```
Comment (ad1992) [inline dev-docs/docs/codebase/json-schema.mdx:6]: We should also document each attribute in a table format for better clarity first and then attach this example below.

**Review by ad1992 — CHANGES_REQUESTED** [review]

File: dev-docs/docs/codebase/json-schema.mdx (hunk 1 above)
Comment (Rajni2002) [reply dev-docs/docs/codebase/json-schema.mdx:6]: Is it something like this
---
Screenshot 2023-07-28 at 3 11 07 PM

**Comment by Rajni2002** [comment]
This is how it's looking now.
---
Screenshot 2023-08-04 at 1 19 17 PM
Screenshot 2023-08-04 at 1 20 01 PM

**Review by Rajni2002 — COMMENTED** [review]
clear

**Comment by dwelle** [comment]
Let's put the `Codebase` section below `@excalidraw/excalidraw`?

**Comment by Rajni2002** [comment]
ok @dwelle

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
I removed the redundant examples as people can check the whole schema example directly, made some tweaks, as well as added clipboard schema. We're good to go, thanks!

**Pull request merged**